source/DSCResources/DSC_SqlSetup/DSC_SqlSetup.psm1

```diff
@@ -1767,6 +1793,13 @@ function Set-TargetResource
     .PARAMETER ProductKey
         Product key for licensed installations.
 
+    .PARAMETER PRODUCTCOVEREDBYSA
+        Specifies the license coverage for SQL Server. True indicates it's covered under Software Assurance or SQL Server subscription.
+        False, or omitting the parameter, indicates it's covered under a SQL Server license.
+        Default value is False.
+
+        Not used in Test-TargetResource.    
```

Comment:
Why is it not checked in `Test-TargetResource`? If the resource is not set correctly should the setup not rerun with the correct parameters?